gl/src/lib.rs

```diff
@@ -607,6 +619,7 @@ impl Device for GLDevice {
         unsafe {
             gl::BindFramebuffer(gl::FRAMEBUFFER, framebuffer.gl_framebuffer); ck();
             gl::Viewport(0, 0, framebuffer.texture.size.x(), framebuffer.texture.size.y()); ck();
+            gl::Scissor(0, 0, framebuffer.texture.size.x(), framebuffer.texture.size.y()); ck();
```

Comment:
Why do you set the scissor box? (I don't think it does anything unless `GL_SCISSOR_TEST` is enabled anyway, no?)